gen/gen_table.go

```diff
@@ -455,30 +457,38 @@ func (conn *ConnPGClient) Update{{ .GoName }}(
 	fieldMask pggen.FieldSet,
 	opts ...pggen.UpdateOpt,
 ) (ret {{ .PkeyCol.TypeInfo.Name }}, err error) {
-	return conn.impl.update{{ .GoName }}(ctx, value, fieldMask)
+	return conn.impl.update{{ .GoName }}(ctx, value, fieldMask, opts...)
 }
 func (p *pgClientImpl) update{{ .GoName }}(
 	ctx context.Context,
 	value *{{ .GoName }},
 	fieldMask pggen.FieldSet,
 	opts ...pggen.UpdateOpt,
 ) (ret {{ .PkeyCol.TypeInfo.Name }}, err error) {
+
```

Comment:
style-nit: one of our linters that we have for the monorepo will complain about leading vertical space in a function like this, and I think I generally agree with that linter. Not a big deal though.